src/init.cpp

```diff
@@ -1018,6 +1023,36 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         return InitError(_("No proxy server specified. Use -proxy=<ip> or -proxy=<ip:port>."));
     }
 
+#if defined(USE_SYSCALL_SANDBOX)
+    if (args.IsArgSet("-sandbox") && !args.IsArgNegated("-sandbox")) {
+        const std::string sandbox_arg{args.GetArg("-sandbox", "")};
+        bool log_syscall_violation_before_terminating{false};
+        if (sandbox_arg == "log-and-kill") {
+            log_syscall_violation_before_terminating = true;
+        } else if (sandbox_arg == "kill") {
```

Comment:
Please add a comment in here, leaving this block empty looks as if something is missing.

Reply:
Makes sense. Fixed!